packages/monaco/src/browser/monaco-editor-model.ts

```diff
@@ -83,6 +84,7 @@ export class MonacoEditorModel implements ITextEditorModel, TextEditorDocument {
         protected readonly resource: Resource,
         protected readonly m2p: MonacoToProtocolConverter,
         protected readonly p2m: ProtocolToMonacoConverter,
+        protected readonly editorPreference: EditorPreferences,
```

Comment:
Changing the signature of a function is a breaking change. In this case, the function is the constructor but the rule still applies. Moving this parameter last and making it optional would not break api. This mean you'd also need a fallback in the logic trying to use it.

Reply:
Typo: missing `s` at the end of the parameter name.